course_discovery/apps/api/tests/test_serializers.py

```diff
@@ -224,6 +224,7 @@ def get_expected_data(cls, course, request):
     def setUp(self):
         super().setUp()
         self.request = make_request()
+        self.edx_org_short_name = 'edX'
```

Comment:
Move this to test case `test_filter_courses_on_edx_org_short_name`

Reply:
Done